### DISCUSSION TITLE
Using devtools with StartLinuxDrm() / framebuffer

### DISCUSSION THREAD

**Discussion by dedounet** [AvaloniaUI/Avalonia#12894] [Q&A] [answered]
Hello everyone! I'm developing an application deployed on a Compute Module 3 + with a Debian Lite distribution without a desktop, there's a screen with a touch screen attached to the system. I can use devtool without any problem on my Windows PC (F12 key) to debug the application, but how do I use it on the machine? 
I found the discussion at [https://github.com/AvaloniaUI/Avalonia/discussions/11748](url) which shows that the keyboard is not supported without the application. If I connect a keyboard, I have access to the console displayed before launching the application but not to my application.

Thanks for your help guys !

**Comment by kekekeks** [comment, marked as answer]
There were plans to get devtools working via HTTP, but we had to spent quite a bit of time fixing them after regressions introduced by a previous "refactoring", so it was never implemented. Maybe later this year we'll try again.

**Reply by dedounet** [reply]
Should I ask for this feature or is it already on the roadmap ?

**Reply by kekekeks** [reply]
It's in a backlog, but isn't listed anywhere, might be worth filing an issue.
